### ISSUE TITLE
Play only turbo and get wrong mmr after game ,without calibration or something 

### ISSUE THREAD

**Issue by rokera24** [ValveSoftware/Dota2-Gameplay#9665]
### Description

Before the update, I was ranked Ancient 5, playing with Immortal, Divine, and Ancient players. However, after the update, I stopped playing ranked games, and Dota assigned me the rank of Legend 3 without any ranked games played. Additionally, it placed me with players of lower MMR. I'm wondering how to resolve this issue or revert back to my previous rank. I attempted to recalibrate, thinking it would fix the problem, but the issue persists.

### Screenshots

![after update](https://github.com/ValveSoftware/Dota2-Gameplay/assets/136167144/7a7faa97-0332-4d55-95ef-398820be36ad)
![beforup](https://github.com/ValveSoftware/Dota2-Gameplay/assets/136167144/3b5e9925-1767-4f7f-9dd8-1061110eb607)
![1234556](https://github.com/ValveSoftware/Dota2-Gameplay/assets/136167144/63b8012c-5966-4a8f-9ad1-fc5f48312b28)

**Comment by rokera24** [comment]
I don't have rank games to generate rank ..., play only turbo for fun

**Comment by Aluerie** [comment]
Tha'ts a dotabuff.com bug, not Dota 2.

**Comment by Aluerie** [comment]
don't duplicate your posts tho #9664

**Comment by rokera24** [comment]
is not  dotabuff bug , i send you picture from there to see the problem how i show you all match before and after

**Comment by Aluerie** [comment]
the medal by dotabuff.com is their bug. Stratz shows correctly that you are uncalibrated.

**Comment by rokera24** [comment]
you cant understand , i know i am uncalibrate , just dota put me with wrong mmr people because i am 2k more than this breket , i work hard to put me in divane games , and after update my game  was with archons and legend

**Comment by rokera24** [comment]
Right, dota has a hidden mmr that determines your level before the rank system, and so hang me how before the update I play with people divan, and after the update only legends and archons let me play
